chroma_core/lib/stratagem.py

```diff
@@ -16,6 +16,19 @@
     "size >= 1t": "greater_than_equal_1t",
 }
 
+labels = {
```

Comment:
It's too bad we need to do this.

Reply:
I'm monitoring https://community.grafana.com/t/how-can-i-map-x-axis-values-to-user-friendly-labels-in-a-series-graph-using-influx/19140. No one has responded yet unfortunately.